packages/web-components/fast-foundation/src/text-field/text-field.template.ts

```diff
@@ -31,12 +31,12 @@ export function textFieldTemplate(
                     })}
                 ></slot>
             </label>
-            <div class="root" part="root">
+            <div class="control" part="control">
                 ${startSlotTemplate(options)}
                 <input
-                    class="control"
-                    part="control"
-                    id="control"
+                    class="input"
+                    part="input"
+                    id="input"
```

Comment:
I'm struggling a bit with this semantic for the parts because as far as users go, the whole thing is the input, we're just using an input to prevent having to recreate an entire input as well as all of the nuanced interactions that go with it. I don't have a better name, but if we decided to go away from this pattern internally I'm not sure input continues to make sense...because again, the whole thing is the "text field" input as far as forms go.

Reply:
Per our discussion we landed on "field".